extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/sources/generated/GeoJsonSourceTest.kt

```diff
@@ -77,24 +75,64 @@ class GeoJsonSourceTest : BaseStyleTest() {
   }
 
   @Test
-  @UiThreadTest
   fun urlTest() {
+    val latch = CountDownLatch(2)
+    val dataList = mutableListOf<String?>()
```

Comment:
```suggestion
    val answerList = mutableListOf<String>()
```

Reply:
it can actually contain `null` value. because Plain json string data getter is not supported and it will return in that case. Although i will update the name.